.github/ISSUE_TEMPLATE/regular-issue.yaml

```diff
@@ -0,0 +1,47 @@
+name: Generic Issue
+description: Open a generic Issue for the OpenJS CPC
+labels:
+  - cross-project-council-agenda
+body:
+  - type: markdown
+    attributes:
+      value: |
+        ### OpenJS Cross Project Council
+
+        Thank you for opening an issue for the OpenJS Cross Project Council.
+
+        **Please remember to follow the Code of Conduct and be respectful.**
+  - type: input
+    id: topic
+    attributes:
+      label: What is this Issue about?
+      description: How can the CPC help you out?
+    validations:
+      required: false
+  - type: textarea
+    id: details
+    attributes:
+      label: Request Details
+      description: Provide us more details about your request to the CPC
+    validations:
+      required: true
+  - type: dropdown
+    id: type
+    attributes:
+      label: Request Type
+      description: What is this Issue about?
+      options:
+        - General Question
+        - Resource Request
+        - CPC Agenda
+        - Legal Request
+    validations:
+      required: true
+  - type: checkboxes
+    id: terms
+    attributes:
+      label: Code of Conduct
+      description: By submitting this issue, you agree to follow our [Code of Conduct](https://github.com/openjs-foundation/cross-project-council/blob/main/CODE_OF_CONDUCT.md)
+      options:
+        - label: I agree to follow the OpenJS Foundation Code of Conduct
+          required: true
```

Comment:
Please, let's not go down the road of turning culture into policy. That would be very sad.

Reply:
The CoC is already policy.

Reply:
Seconding @ljharb here. I find it useful for reminding people about our CoC because some are just not aware of it. I could remove the checkbox of course if it feels too much tho.

Reply:
Don't get me wrong, policy is important. But it really ought to only be there as a backup for when culture fails. If you put policy front and center from the get go, you've already lost.

Reply:
Why would've already lost? Can you explain? 😅 it is a norm to on every front-page to show what is the CoC, otherwise how would newcomers know what is the CoC?

I just find it interesting to have the CoC being mentioned at all, but I can definitely remove the checkboxes :)

Reply:
I see Tobie's point. It's like reprimanding before they've done something wrong. I support having a link to the CoC, but I'm +1 for removing the checkbox.

Reply:
I really don't agree; I think that culture is guided by policy, and if you aren't putting your policy front and center then you're *ensuring* your culture fails.

Reply:
I think it's still front and center without the checkbox